src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -33,22 +35,31 @@ export default function ConfirmedAppointmentDetailsPage() {
   );
   const appointmentDate = moment.parseZone(appointment?.start);
 
-  useEffect(() => {
-    dispatch(fetchConfirmedAppointmentDetails(id, 'va'));
+  const isVideo = appointment?.vaos?.isVideo;
+  const isCommunityCare = !!match.path.includes('cc');
+  const isVA = !!match.path.includes('va');
```

Comment:
```suggestion
  const isVA = !isVideo && !isCommunityCare;
```

Reply:
Just my 2 cents but ```const isVA = !!match.path.includes('va');``` seems clearer to me.

Reply:
> Just my 2 cents but `const isVA = !!match.path.includes('va');` seems clearer to me.

Yeah, it does seem to read a little awkward but because video appointments are also VA appointments, its detailed description shows up twice in the appointment details page because it passes `isVideo` and `isVA` if the latter is based on the URL path. 
```
{isVideo && (
        <DetailsVideo appointment={appointment} facilityData={facilityData} />
      )}
{isVA && (
        <DetailsVA
          appointment={appointment}
          facilityData={facilityData}
          useV2={useV2}
        />
```

While it is a round about way of testing for `isVA`, it excludes video appointments.